_includes/contact_form.html

```diff
@@ -13,6 +13,7 @@
     case "Changing Settings":
       document.getElementById("sub-topic").options[0] = new Option("{% t contact_page.support_request_form.subtopics.none %}", "");
       document.getElementById("sub-topic").options[1] = new Option("{% t contact_page.support_request_form.subtopics.authentication_app_setup %}", "Authentication app setup");
+      document.getElementById("sub-topic").options[2] = new Option("{% t contact_page.support_request_form.subtopics.backup_codes %}", "Backup codes");
```

Comment:
We'll want to test to make sure forms submitted via these options show up correctly in Salesforce. I think we _might_ have to do it once this is live as I don't believe any form submissions worked from our preview builds in the past.

Reply:
Will check with the contact center folks.

Reply:
They asked to let them know when this is deployed to production and said they will be adding the relevant field in SF.